src/core/forces_cabana.hpp

```diff
@@ -114,7 +114,7 @@ struct ForcesKernel {
     auto const dist = d.norm();
 
 #if defined(ESPRESSO_EXCLUSIONS) or defined(ESPRESSO_DPD) or                   \
-    defined(ESPRESSO_DIPOLES)
+    defined(ESPRESSO_THOLE) or defined(GAY_BERNE) or defined(ESPRESSO_DIPOLES)
```

Comment:
here and elsewhere, don't forget to add the `ESPRESSO_` prefix to all ESPResSo features :-)

Reply:
I added `ESPRESSO_` prefix.

Reply:
Good catch! Sorry about that. I have fixed it.